docs/data_providers/data_format/summary.md

```diff
@@ -52,10 +52,13 @@ tbody tr td:first-child {
 This block provides a set of core details for the dataset. You can only provide
 a single value for each field.
 
-* **SAFE Project ID**: This is the project number from the SAFE project website.
-  When you upload your dataset, you will also be asked to choose a project for
-  your dataset: these two numbers must match. Note that you can only upload data
-  to a project of which you are a member.
+* **Project ID**: This is only used if the organisation you are uploading to uses
+  projects to group datasets. If they are then you should obtain relevant project IDs
+  from the organisation's data manager. Then when you upload your dataset, you will also
+  be asked to choose a project for your dataset: these two numbers must match. Note that
+  you can only upload data to a project of which you are a member. The legacy name for
+  this field is **SAFE Project ID**, using this name will still work but is not the
+  preferred option.
```

Comment:
Some of the text relates to the existing SAFE website upload page is not generally relevant

```suggestion
* **Project ID**: This is only used if the organisation you are uploading to uses
  projects to group datasets. If they are then you should obtain relevant project IDs
  from the organisation's data manager and add them in your Summary worksheet. 
  The legacy name for this field is `SAFE Project ID`: using this name will still work 
  but is not the preferred option.
```